docs/src/getting-started/running-linuxcnc.adoc

```diff
@@ -140,30 +140,30 @@ available in other GUIs as well.
 The Configuration Selector copies all files needed
 for a configuration to a new subdirectory of ~/linuxcnc/configs
 (equivalently: /home/username/linuxcnc/configs).  Each
-created directory will include at least one ini file (iniflename.ini)
+created directory will include at least one INI file (iniflename.ini)
 that is used to describe a specific configuration.
 
 File resources within the copied directory will typically
-include one or more ini file (filename.ini) for related
+include one or more INI file (filename.ini) for related
 configurations and a tool table file (toolfilename.tbl).
-Additionally, resources may include halfiles (filename.hal,
+Additionally, resources may include HAL files (filename.hal,
 filename.tcl), a README file for describing the directory, and
 configuration specific information in a text file named after
 a specific configuration (inifilename.txt).  That latter two
 files are displayed when using the Configuration Selector.
 
 The supplied sample configurations may specify HALFILEs
```

Comment:
```suggestion
The supplied sample configurations may specify HAL files
```

Reply:
I was not sure about this one, thought "HALFILE" was a configuration parameter and should not be changed.

Reply:
Yes it is more or less but how it was originally written it was not really clear.
We could extend it to "The supplied sample configurations may specify HAL files as parameter `HALFILE=`in the [HAL] section". But I think that would be too much here.